renamed editBox coponent to MentorEditLearningObj

SULAIMAN-100 opened the pull request:
…ued code lines

Before sending your Pull Request, be sure to check that you are making the pull request from your branch to the dev branch! Please do not try to push directly into branches on the main repo.




renamed editBox coponent to MentorEditLearningObj and cleaned some unused code lines

AltomHussain committed:
Merge pull request #97 from NawalC/dev

Dev

Iman-feisali committed:
Merge pull request #98 from NawalC/dev

fixed image issue on login page

AltomHussain committed:
Merge pull request #99 from NawalC/dev

Readme

AltomHussain committed:
Replaced all options with the map

AltomHussain committed:
Removed all logs

petemcfarlane committed:
Add initial deploy-staging circleci workflow

SULAIMAN-100 committed:
renamed editBox coponent to MentorEditLearningObj and cleaned some unued code lines

petemcfarlane commented on client/src/App.css:
```diff
@@ -214,14 +214,14 @@ li {
 } */
-
+/* 
 .skills-container .edit-btn:not A:hover {
   border-radius: 4px;
   color: white;
   text-decoration: none;
   border-bottom: 1px solid whites;
   text-align: center;
-}
+} */
```
Is it worth deleting this, rather than simply commenting out?

petemcfarlane reviewed (approved):
Looks good, much clearer name 👍

AltomHussain committed:
Remaned BtnContainer to ProgressTrackingButtons

AltomHussain committed:
Added cookieSession Library

AltomHussain committed:
Added cookie session in the login and register api

AltomHussain committed:
refactoring(removed user from the req and used req.session.user instead)

petemcfarlane committed:
Merge pull request #1 from CodeYourFuture/feature/deploy-staging

Add initial deploy-staging circleci workflow

AltomHussain committed:
Merge branch 'dev' into Refactor-AddForm

AltomHussain committed:
Remove all jwt tokens from some files where is not required

AltomHussain committed:
Refactored the MentorEdit component

SULAIMAN-100 committed:
Merge pull request #7 from CodeYourFuture/Refactor-mentor-edit

Refactor mentor edit

SULAIMAN-100 committed:
Merge pull request #4 from CodeYourFuture/Refactor-AddForm

Refactor add form

SULAIMAN-100 committed:
resolved the conflects

SULAIMAN-100 committed:
merged remove-jwt-tokens branch into dev

SULAIMAN-100 committed:
modified logo to be clickable used to back to the default page of the user

SULAIMAN-100 committed:
removed useState and useEffect

SULAIMAN-100 committed:
added the first stage of github authintication

AltomHussain committed:
altered user table(added github_id)

AltomHussain committed:
Added some changes in api.js(create githubAuth endpoint for login)

AltomHussain committed:
changed the logout flow(removed the logout icon from all files and just
used in header.js also created logout endpoin in api.js)

AltomHussain committed:
updated the mentorsView page(extracted the useQuery into reusable file
and imported it into the mentors view page)

AltomHussain committed:
updated the logingForm.js(removed the emial and password fields from the
front-end), Updated the api.js(Added the github user name inside
/githubAuth endpoind added pseudo code in the register endpoind to work
on in the next step"

AltomHussain committed:
Displayed the user name in the front-end from the DB instead of the
localstorage,

AltomHussain committed:
got the username form the db instead of the localstorage

AltomHussain committed:
used the file for testing

AltomHussain committed:
Removed the login form and its functionality and only left a button to login

AltomHussain committed:
Replaced password with the gitId

AltomHussain committed:
Removed password validation

AltomHussain committed:
Added axios package

AltomHussain committed:
Replaced password with github_id and removed the login endpoint

AltomHussain committed:
Removed unused code

AltomHussain committed:
added and removed log

AltomHussain committed:
Created the reuable component for uselocation

AltomHussain committed:
Styled the Btn in the login form

AltomHussain committed:
added css for the login btn

AltomHussain committed:
removed commented code

SULAIMAN-100 committed:
fixed logo link path

SULAIMAN-100 committed:
added bootstrap navbar

SULAIMAN-100 committed:
final version of the logo

AltomHussain committed:
Merge pull request #8 from CodeYourFuture/clickable-logo

modified logo to be clickable used to back to the default page of the…

AltomHussain committed:
got the the user name from the verify end-point and deisplayed on the screen instead of using the localstorage

AltomHussain committed:
Removed the the usage of the localstorage

AltomHussain committed:
Made the useState initiacl val to null so the data will be fetched correctl!

AltomHussain committed:
Removed the functionality of clearing the localStorage

AltomHussain committed:
Removed the usage of localstorage

AltomHussain committed:
Removed the usage of the localStorage

AltomHussain committed:
Removed the usage of the  localStorage

AltomHussain committed:
Removed the userid from the learningobjectives endpoint

AltomHussain committed:
Adjusted the cdn

SULAIMAN-100 committed:
added bootstrap navbar

Commit:
removed button and removed spaces

Commit:
checked bootstrap version - discussion needed

AltomHussain committed:
Merge branch 'dev' into bootstrap-navbar

AltomHussain committed:
Merge pull request #10 from CodeYourFuture/bootstrap-navbar

Bootstrap navbar

Commit:
created select options for city and class in progress

Iman-feisali committed:
we changed the initial state of the user name from null to empty object

Commit:
created city object and looped them

AltomHussain committed:
Removed the usage of jwt from the server side

AltomHussain committed:
fixed conflects

AltomHussain committed:
updated based on the feedback

Commit:
added styling to select fields

Commit:
completed if block and added all the locations

AltomHussain committed:
Merge pull request #11 from CodeYourFuture/signup-dropdown

created select options for city and class in progress

Commit:
setup github locally

Commit:
merged github-signUp to dev

Commit:
got update and merged again

Commit:
resolved conflicts

Commit:
resolved more conflicts

AltomHussain committed:
Merge pull request #12 from CodeYourFuture/signup-dropdown

completed if block and added all the locations

Commit:
added axios to dependencies

Commit:
fixing current deployment issues

Commit:
fixing deployment

Commit:
created a callback to fetch client id

SULAIMAN-100 committed:
Merge branch 'master' of https://github.com/CodeYourFuture/knowledge-checklist into deselect-btns

SULAIMAN-100 committed:
created deselect endpoint

SULAIMAN-100 committed:
added deselect function

SULAIMAN-100 committed:
modified CSS

SULAIMAN-100 committed:
created deselect function to update ability

Commit:
created dir for learning obj data

Commit:
created db tables using node js

Commit:
created pg query using db config

SULAIMAN-100 committed:
added highlight student feature

SULAIMAN-100 committed:
fixed deselect logic

SULAIMAN-100 committed:
fixed deselect logic2

SULAIMAN-100 committed:
modified css

AltomHussain committed:
added git React-client-github-id

AltomHussain committed:
Cleared all logs and sorted the security of the mentorEditLearningObj component

Commit:
added db migration scripts to staging

Commit:
added db migration scripts for production

AltomHussain committed:
Deleted the deselect endpoint and amended set ability using Pete's suggestion

NawalC committed:
Merge pull request #15 from CodeYourFuture/secure-learningOb-page

Cleared all logs and sorted the security of the mentorEditLearningObj…

NawalC committed:
Merge pull request #13 from CodeYourFuture/deselect-btns

Deselect btns

NawalC committed:
Merge pull request #14 from CodeYourFuture/highlight-student

Highlight student

Commit:
corrected spelling error of user_github

Commit:
added column cyf_city

Commit:
corrected spelling errors

Commit:
new migration script to add a new column and rename column

Commit:
fixed error msg by added a return in /gitAuth

AltomHussain committed:
Merge pull request #16 from CodeYourFuture/db-migrate

Db migrate

AltomHussain committed:
identified ununderstandable issue

Commit:
edited github ud variable to fix staging link

AltomHussain committed:
Merge pull request #17 from CodeYourFuture/db-migrate

fixing heroku error

Commit:
still trying to fix staging link id undefined

AltomHussain committed:
Merge pull request #18 from CodeYourFuture/db-migrate

still trying to fix staging link id undefined

Commit:
added learning objectives from cyf syllabus

AltomHussain committed:
Merge branch 'dev' of https://github.com/CodeYourFuture/knowledge-checklist into dev

AltomHussain committed:
Updated the ReadME.md file

Commit:
added learning objectives inline with syllabus

AltomHussain committed:
delete the id from

AltomHussain committed:
Merge pull request #19 from CodeYourFuture/addlearningobjective

Addlearningobjective

AltomHussain committed:
Merge branch 'dev' of https://github.com/CodeYourFuture/knowledge-checklist into dev

AltomHussain committed:
Modified the desect

AltomHussain committed:
Merge pull request #21 from CodeYourFuture/deselect-btns

Deselect btns

AltomHussain committed:
Merge branch 'dev' of https://github.com/CodeYourFuture/knowledge-checklist into dev

SULAIMAN-100 committed:
Merge pull request #20 from CodeYourFuture/update-doc-md

Update doc md

AltomHussain committed:
Merge branch 'dev' of https://github.com/CodeYourFuture/knowledge-checklist into dev

AltomHussain committed:
Resolved conflicts
